### PR TITLE
Working on coverage fix.

### PR DESCRIPTION
**Pull request by adunndevster**

### CODE REVIEW

File: docs-markdown/src/test/suite/index-coverage.ts [hunk 1]
```diff
@@ -31,9 +34,7 @@ function setupNyc() {
 }
 
 export function run(): Promise<void> {
-
-	const nyc = setupNyc();
-
+	let nyc = setupNyc();
```
Comment (bharney0) [inline docs-markdown/src/test/suite/index-coverage.ts:37]: Do we need this `nyc` if its also at the bottom there inside `mocha.run`?

File: docs-markdown/src/test/suite/index-coverage.ts [hunk 2]
```diff
@@ -61,18 +62,18 @@ export function run(): Promise<void> {
 					if (failures > 0) {
 						e(new Error(`${failures} tests failed.`));
 					} else {
+						nyc = setupNyc();
```
Comment (adunndevster) [inline docs-markdown/src/test/suite/index-coverage.ts:65]: Running this again here correctly wraps the tested code files with nyc magic, and then spits out an _accurate_ report.

File: docs-markdown/src/test/suite/index-coverage.ts (hunk 1 above)
Comment (adunndevster) [inline docs-markdown/src/test/suite/index-coverage.ts:37]: Settung up nyc here sets up the initial framework of files in the out folder. Without this, line 65 below crashes.

File: docs-markdown/src/test/suite/index-coverage.ts [hunk 3]
```diff
@@ -22,6 +24,7 @@ function setupNyc() {
 		reporter: ["text", "html", "cobertura"],
 		require: [
 			"ts-node/register",
+			"source-map-support/register",
```
Comment (bharney0) [inline docs-markdown/src/test/suite/index-coverage.ts:27]: Do we need to add the `source-map-support` package?

File: docs-markdown/src/test/suite/index-coverage.ts (hunk 1 above)
Comment (adunndevster) [reply docs-markdown/src/test/suite/index-coverage.ts:37]: @bharney0 - See this to answer your question about needing this line.

**Pull request merged**